tests/testcases/core/db_classes/EE_Attendee_Test.php

```diff
@@ -74,7 +74,7 @@ public function test_save_and_clean_billing_info_for_payment_method(){
 				'phone'		=> '9991231234',
 				'credit_card'=>'4007000000027',
 				'exp_month'=>'12',
-				'exp_year'=>'2020',
+				'exp_year'=>'2030',
```

Comment:
You will need to update it again then 😄

Reply:
dang it... you're right!!! will change it to 3030 then and hope I still have that same problem